src/NetAppFiles/NetAppFiles/Account/UpdateNetAppFilesAccount.cs

```diff
@@ -0,0 +1,84 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System.Collections;
+using System.Management.Automation;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Commands.NetAppFiles.Common;
+using Microsoft.Azure.Commands.NetAppFiles.Models;
+using Microsoft.Azure.Management.NetApp;
+using Microsoft.Azure.Management.NetApp.Models;
+using Microsoft.Azure.Commands.NetAppFiles.Helpers;
+
+namespace Microsoft.Azure.Commands.NetAppFiles.Account
+{
+    [Cmdlet(
+        "Update",
+        ResourceManager.Common.AzureRMConstants.AzureRMPrefix + "NetAppFilesAccount",
+        SupportsShouldProcess = true,
+        DefaultParameterSetName = FieldsParameterSet), OutputType(typeof(PSNetAppFilesAccount))]
+    [Alias("Update-AnfAccount")]
+    public class UpdateAzureRmNetAppFilesAccount : AzureNetAppFilesCmdletBase
+    {
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The resource group of the ANF account")]
+        [ValidateNotNullOrEmpty]
+        [ResourceGroupCompleter()]
+        public string ResourceGroupName { get; set; }
+
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The name of the ANF account")]
+        [ValidateNotNullOrEmpty]
+        [Alias("AccountName")]
+        public string Name { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "The location of the resource")]
+        [ValidateNotNullOrEmpty]
+        [LocationCompleter("Microsoft.NetApp/netAppAccounts")]
+        public string Location { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            HelpMessage = "A hashtable array which represents the active directories")]
+        [ValidateNotNullOrEmpty]
+        public Hashtable[] ActiveDirectories { get; set; }
```

Comment:
What does this cmdlet allow you to change about the resource?  Also, same parameter name and parameter set comments